nbconvert/exporters/tests/test_kernel_templates.py

```diff
@@ -0,0 +1,27 @@
+"""Tests for Kernel-specific templates"""
+
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+from test.support import EnvironmentVarGuard
```

Comment:
https://docs.python.org/3/library/test.html

> The test package is meant for internal use by Python only. It is documented for the benefit of the core developers of Python. Any use of this package outside of Python’s standard library is discouraged as code mentioned here can change or be removed without notice between releases of Python.

We already require testpath for tests, so you can use [testpath.modified_env](http://testpath.readthedocs.io/en/latest/env.html#testpath.modified_env) to achieve this.